src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -132,12 +132,15 @@
 
     private final long ENTITY_SETS_REFRESH_INTERVAL = 15 * 60 * 1000; // 15 minutes
 
+    // orgId -> participantId -> EKID
+    private final Map<UUID, Map<String, UUID>> studyParticipants = Maps.newHashMap();
```

Comment:
This structure basically means that participant ids HAVE to be unique across studies, which doesn’t seem ideal. We've already seen researchers using this pattern... assigning the same participant id to the same human but in different studies.

Reply:
Right!!! Will change.